src/e2e-test/java/io/cdap/plugin/utils/State.java

```diff
@@ -0,0 +1,419 @@
+/**
+ * Autogenerated by Avro
+ * <p>
+ * DO NOT EDIT DIRECTLY
+ */
+package io.cdap.plugin.utils;
+
+import org.apache.avro.message.BinaryMessageDecoder;
+import org.apache.avro.message.BinaryMessageEncoder;
+import org.apache.avro.message.SchemaStore;
+import org.apache.avro.specific.SpecificData;
+import org.apache.avro.util.Utf8;
+
+/** A list of states in the United States of America. */
+@org.apache.avro.specific.AvroGenerated
+public class State extends org.apache.avro.specific.SpecificRecordBase
+  implements org.apache.avro.specific.SpecificRecord {
+  public static final org.apache.avro.Schema SCHEMA = new org.apache.avro.Schema.Parser()
+    .parse("{\"type\":\"record\",\"name\":\"State\",\"namespace\":\"utilities\",\"doc\":\"A list of states in the" +
+             " United States of America.\",\"fields\":[{\"name\":\"name\",\"type\":\"string\",\"doc\":\"The common " +
+             "name of the state.\"},{\"name\":\"post_abbr\",\"type\":\"string\",\"doc\":\"The postal code " +
+             "abbreviation of the state.\"}]}");
+  private static final long serialVersionUID = -6098929419967278282L;
+  private static final SpecificData MODEL$ = new SpecificData();
+  private static final BinaryMessageEncoder<State> ENCODER =
+    new BinaryMessageEncoder<>(MODEL$, SCHEMA);
+  private static final BinaryMessageDecoder<State> DECODER =
+    new BinaryMessageDecoder<>(MODEL$, SCHEMA);
+  @SuppressWarnings("unchecked")
+  private static final org.apache.avro.io.DatumWriter<State>
+    WRITER$ = (org.apache.avro.io.DatumWriter<State>) MODEL$.createDatumWriter(SCHEMA);
+  @SuppressWarnings("unchecked")
+  private static final org.apache.avro.io.DatumReader<State>
+    READER$ = (org.apache.avro.io.DatumReader<State>) MODEL$.createDatumReader(SCHEMA);
+  /** The common name of the state. */
+  private CharSequence name;
+  /** The postal code abbreviation of the state. */
+  private CharSequence postabbr;
+
+  /**
+   * Default constructor.  Note that this does not initialize fields
+   * to their default values from the schema.  If that is desired then
+   * one should use <code>newBuilder()</code>.
+   */
+  public State() {
+  }
+
+  /**
+   * All-args constructor.
+   * @param name The common name of the state.
+   * @param postabbr The postal code abbreviation of the state.
+   */
+  public State(CharSequence name, CharSequence postabbr) {
```

Comment:
use naming convention for variable CharSequence correctly. Change it to something more relevant.

Reply:
same comment as above.